buildFull.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/bash
 ./gradlew clean downloadUrlDependencies build shadowJar "$@"
+
```

Comment:
For developers who are rebuilding the code, helpful to copy our kafka 0.10.0.1 library over each time